docs/B-Computations/types.md

```diff
@@ -24,459 +31,492 @@ This chapter describes the four most common types in the C language and the rang
 
 The four most common types of the C language for performing arithmetic calculations are:
 
-* char
-* int
-* float
-* double
+* `char`
+* `int`
+* `float`
+* `double`
 
 A **`char`** occupies one byte and can store a small integer value, a single character or a single symbol: 
-
-<table border="0">
-<tr><td align="center" colspan="8">char</td></tr>
-<tr><td align="center" colspan="8">1 Byte</td></tr>
-<tr><td bgcolor="#ffffcc">&nbsp;</td>
-    <td bgcolor="#ffffcc">&nbsp;</td>
-    <td bgcolor="#ffffcc">&nbsp;</td>
-    <td bgcolor="#ffffcc">&nbsp;</td>
-    <td bgcolor="#ffffcc">&nbsp;</td>
-    <td bgcolor="#ffffcc">&nbsp;</td>
-    <td bgcolor="#ffffcc">&nbsp;</td>
-    <td bgcolor="#ffffcc">&nbsp;</td></tr>
-</table>
-
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 800 165">
+    <foreignObject x="33%" width="110%" height="110%">
+        <body xmlns="http://www.w3.org/1999/xhtml">
+            <table border="0">
+            <tr><td align="center" colspan="8"><code>char</code></td></tr>
+            <tr><td align="center" colspan="8">1 Byte</td></tr>
```

Comment:
Are all texts intended to be centered? 
I see that the top part of this file is left-aligned

Reply:
I think the tables representing the memory look better with the text centered:
![image](https://user-images.githubusercontent.com/43410858/142678956-04feda15-38dc-4394-aaa0-071e639d7efd.png)

As for the other tables on the page, I don't know what's the best approach for alignment. The original document has everything aligned center. I was thinking of maybe aligning it in an 'MS Excel' style, with texts align left and numbers align right:
![image](https://user-images.githubusercontent.com/43410858/142680864-70d2235c-99a8-4074-a832-1a62952564ab.png)